dkc/core/tests/test_authorized_uploads.py

```diff
@@ -0,0 +1,179 @@
+from datetime import timedelta
+
+from django.utils import timezone
+import pytest
+
+from dkc.core.models import AuthorizedUpload
+
```

Comment:
@brianhelba @jbeezley We should do a final holistic pass, once the implementation is frozen, to ensure that the important cases are getting covered by these tests. I'm sure the coverage report will be helpful.

Reply:
Yes, I've been using it in development to check myself. It's very nice.